exasol_integration_test_docker_environment/lib/test_environment/spawn_test_database.py

```diff
@@ -160,20 +161,24 @@ def _connect_docker_network(
         docker_client: DockerClient,
         container: Container,
         ip_address: str,
-    ):
+    ) -> None:
         network = docker_client.networks.get(self.network_info.network_name)
         aliases = self._get_network_aliases()
         network.connect(container, ipv4_address=ip_address, aliases=aliases)
 
-    def _port_mapping(self, internal_ports, forwarded_ports):
+    def _port_mapping(
+        self, internal_ports, forwarded_ports
```

Comment:
are the missing types intended

Reply:
Will be fixed in https://github.com/exasol/integration-test-docker-environment/issues/453